{"name": "Проверяем, что нельзя создать курьера с существующим логином, запрос возвращает ошибку с кодом 409", "status": "passed", "start": 1727696242772, "stop": 1727696243067, "uuid": "697b7ddb-7524-43c7-8ec5-c6692785db1c", "historyId": "9c1462b9d23eb2fd2dc889d029ad3343", "testCaseId": "9c1462b9d23eb2fd2dc889d029ad3343", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_with_existing_login_error_409", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "hanastasia-mbby"}, {"name": "thread", "value": "14322-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}